kustomize/README.md

```diff
@@ -0,0 +1,42 @@
+# OpenShift configuration with Kustomize
+
+Resource configuration and secret consumption for OpenShift projects are managed
+using [`kustomize`](https://kubectl.docs.kubernetes.io/guides/introduction/kustomize/).
+
+## Setup
+
+1. Populate `service/secrets/*`.
+
+    Sensitive values and files are located in the
+    [Office Hours Secrets](https://www.dropbox.com/sh/n1igrgdsm4rt4uf/AAAXLbZOT7tpVk8XZEQj5E0ca?dl=0)
+    Dropbox folder. Merge the `base` and `overlays` directories with their equivalents in the `service`
+    directory in your local repository.
+
+2. Install `kustomize`.
+
+    You can install Kustomize using the command from the
+    [website](https://kubectl.docs.kubernetes.io/installation/kustomize/binaries/),
+    and adding version 3.8.5 (what is supported for now) as an argument.
+    ```
+    curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash -s 3.8.5
```

Comment:
the kustomize folder is already included in this PR. So when I run this command, I got the following error:

/Users/zqian/dev_cae/canvas-app-explorer/kustomize exists and is a directory. Remove it first.

Reply:
When I do this command in my home folder, I got the following error:
`➜  ~ curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash -s 3.8.5
Version v3.8.5 does not exist.`

Update the version number to 4.5.5 works:

`➜  ~ curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash -s 4.5.5
{Version:kustomize/v4.5.5 GitCommit:daa3e5e2c2d3a4b8c94021a7384bfb06734bcd26 BuildDate:2022-05-20T20:25:40Z GoOs:darwin GoArch:arm64}
kustomize installed to /Users/zqian/kustomize`

Reply:
Thanks I'll try this new version. 3.8.5 installed for me and I just copied this template from ROHQ. 

You'd have to add this to your path too or call it as `~/kustomize` as I think it just downloads it to your home directory by default.

Reply:
4.5.5 worked fine for me. I think the issue was probably that it didn't work on ARM for the older version.